addons/web/static/src/views/graph/graph_renderer.js

```diff
@@ -498,14 +631,18 @@ export class GraphRenderer extends Component {
             },
             ticks: {
                 callback: (value) => this.formatValue(value, allIntegers),
-                color:
-                    cookie.get("color_scheme") === "dark"
-                        ? getColor(15, cookie.get("color_scheme"))
-                        : null,
+                color: GRAPH_LABEL_COLOR,
             },
-            suggestedMax: 0,
-            suggestedMin: 0,
             stacked: mode === "line" && stacked ? stacked : undefined,
+            grid: {
+                display: mode === "line" ? false : true,
+                color: GRAPH_GRID_COLOR,
+            },
+            border: {
+                display: false,
+            },
+            suggestedMax: getMinMaxValue(true, this.model.data, stacked),
```

Comment:
getMinMaxValue should return both min and max in an object and here you could deconstruct the object. For now you compute min and max 2 times each